tests/tests.py

```
@@ -7,11 +7,18 @@
 from snakemake_interface_executor_plugins.settings import ExecutorSettingsBase
 from unittest.mock import MagicMock, patch
 import pytest
+import tempfile
+import yaml
+from pathlib import Path
```

Ruff (0.14.2)

Redefinition of unused `Path` from line 4

Remove definition: `Path`

(F811)